pkg/virt-api/rest/spice.go

```diff
@@ -74,9 +68,11 @@ func spiceFromVM(vm *v1.VirtualMachine) (*v1.Spice, error) {
 				Host: d.Host,
 				Port: d.Port,
 			}
-			if spiceProxy != "" {
-				spice.Info.Proxy = fmt.Sprintf("http://%s", spiceProxy)
+
+			if proxy, ok := config.Get("spice-proxy"); ok {
+				spice.Info.Proxy = fmt.Sprintf("http://%s", proxy)
```

Comment:
I think that it also makes sense to place the protocol also in the value to allow http and https. What do you think?

Reply:
The current patch has feature parity with what we have on the cli if I am not mistaken, thus to me it would be fine to merge this patch.